src/Microsoft.Sbom.Adapters/Adapters/ComponentDetection/ExtendedScannedComponent.cs

```diff
@@ -35,12 +35,22 @@ public ScannedComponentWithLicense(ScannedComponent? other = null)
     }
 
     /// <summary>
-    /// Gets or sets the license.
+    /// Gets or sets the license concluded.
     /// </summary>
-    public string? License { get; set; }
+    public string? LicenseConcluded { get; set; }
 
     /// <summary>
-    /// Converts a <see cref="ScannedComponentWithLicense" /> to an <see cref="SbomPackage" />.
+    /// Gets or sets the license declared (This is what is found in the package files).
+    /// </summary>
+    public string? LicenseDeclared { get; set; }
```

Comment:
I think the documentation for concluded needs to be updated to reflect where it comes from and how it is different from declared.